hydra-node/json-schemas/logs.yaml

```diff
@@ -1432,6 +1433,8 @@ definitions:
         type: array
         items:
           $ref: "api.yaml#/components/schemas/Transaction"
+      allTxs:
+        type: object
```

Comment:
REVIEW: this is a `Map (TxIdType tx) tx`
does it makes sense to break this type down into a more granular spec?

Reply:
It's quite internal, but the other fields of the `CoordinatedHeadState` are also specified, so I would say "yes because of consistency"

We should have `TxId` and `Transaction` schemas